include/chemfiles/capi/atom.h

```diff
@@ -183,7 +183,7 @@ CHFL_EXPORT chfl_status chfl_atom_atomic_number(const CHFL_ATOM* atom, uint64_t*
 /// @return The operation status code. You can use `chfl_last_error` to learn
 ///         about the error if the status code is not `CHFL_SUCCESS`.
 CHFL_EXPORT chfl_status chfl_atom_set_property(
-    CHFL_ATOM* atom, const char* name, const CHFL_PROPERTY* property
+    CHFL_ATOM* const atom, const char* name, const CHFL_PROPERTY* property
```

Comment:
having a const here (in the declaration) is actually useless, as it will not be checked by the compiler.